wcivf/apps/people/models.py

```diff
@@ -153,6 +154,70 @@ def get_max_facebook_ad_spend(self):
             )
         )
 
+    @property
+    def facebook_personal_username(self):
+        facebook_personal_url = self.facebook_personal_url
+        facebook_split = list(filter(None, facebook_personal_url.split("/")))
+        facebook_personal_username = facebook_split[-1]
+
+        return facebook_personal_username
+
+    @property
+    def facebook_username(self):
+        facebook_page_url = self.facebook_page_url
+        facebook_split = list(filter(None, facebook_page_url.split("/")))
+        facebook_username = facebook_split[-1]
+
+        return facebook_username
+
+    @property
+    def instagram_username(self):
+        instagram_url = self.instagram_url
+        instagram_split = list(filter(None, instagram_url.split("/")))
+        instagram_username = instagram_split[-1]
+
+        return instagram_username
+
+    @property
+    def linkedin_username(self):
+        linkedin_url = self.linkedin_url
+        linkedin_split = list(filter(None, linkedin_url.split("/")))
+        linkedin_username = linkedin_split[-1]
+
+        return linkedin_username
+
+    @property
+    def youtube_username(self):
+        youtube_url = self.youtube_profile
+        if ("channel") in youtube_url:
```

Comment:
You don't strictly need the parenthesis around `"channel"` when you are checking if a subset exists within a string